teemaderegister-react/src/components/addNewUserForm.jsx

```diff
@@ -1,111 +0,0 @@
-import React, {useEffect, useState} from 'react'
```

Comment:
merged to master the change that removed the file, should be also ok if the removal is in this PR, just letting you know